microsoftgraph/msgraph-cli-archived
# Fix/ctr+c

## Overview
Add support for killing the CLI process with **CTRL+C**

### Demo

N/A

### Notes

N/A

## Testing Instructions

* Pull these changes
* Run `mg login --scopes user.read`
* Press **CTRL + C**
* Observe that the process is terminated

Fixes #63
